managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1869,6 +1876,10 @@ private void maybeOffload(CompletableFuture<PositionImpl> finalPromise) {
                 });
 
             long threshold = config.getOffloadAutoTriggerSizeThresholdBytes();
```

Comment:
Can we delete `offloadAutoTriggerSizeThresholdBytes` and `offloadLedgerDeletionLagMs` in  `ManagedLedgerConfig`? since there is `OffloadPolicies` in `LedgerOffloader`.

Reply:
@codelipenghui Thanks for the review. @sijie wanted to keep default configuration on `ManagedLedgerConfig` while `OffloadPolicies` in `LedgerOffloader` are not defined for current namespace. WDYT?

Reply:
@KannarFr  We already keep `offload_threshold `  and `offload_deletion_lag_ms ` in Policies, why  need to also keep `offloadAutoTriggerSizeThresholdBytes ` and `offloadAutoTriggerSizeThresholdBytes ` in managedldger config? I still don't understand, why need to keep them in managedldger config.

Reply:
Got it, you are right, fixing...